Example/Example/SampleContent/underline.html

```diff
@@ -2,3 +2,4 @@
 <p><span style="text-decoration: underline;">Alternative&nbsp;underline&nbsp;text</span></p>
 <p>Before link <a href="www.wordpress.com">Link&nbsp;to&nbsp;WordPress</a></p>
 <p>Before link <a href="www.jetpack.com">Link&nbsp;to&nbsp;Jetpack</a> After link</p>
+Link at end of text<a href="www.jetpack.com">Link&nbsp;to&nbsp;Jetpack</a>
```

Comment:
Is it relevant for the test that the las line is not in a `p` tag and `text` and `Link to Jetpack` are together?

Reply:
Yes, because it needs not to have any extra new lines for the bug to be visible.